test/pummel/test-net-timeout.js

```diff
@@ -79,7 +79,7 @@ process.on('exit', function() {
   assert.ok(starttime != null);
   assert.ok(timeouttime != null);
 
-  diff = timeouttime - starttime;
+  var diff = timeouttime - starttime;
```

Comment:
`make test-pummel` shows this error (along with some others). It seems these tests aren't ran as part of `make test` at all.